### PR TITLE
Cleaned up documentation for installation process

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | #2370 
| License       | MIT

`install.sh` is back! But it's simpler. 

You can now install wallabag with only 3 commands: 

```
git clone https://github.com/wallabag/wallabag.git
cd wallabag
./install.sh
```

We won't need to update wallabag for each new release with the new tag.

### CODE REVIEW

**Comment by nicosomb** [comment]
It's a quick draft. Today, we have so many pages which tell us how to install wallabag. I wanted to standardize the installation process. 

I need to review the update process. But I want your opinion about this first draft. 

If it's OK, I also need the help of german translators in this PR (ping @jlnostr @Strubbl).

File: README.md [hunk 1]
```diff
@@ -6,23 +6,23 @@
...
 Then you can install wallabag by executing the following commands:
 
 ```
     git clone https://github.com/wallabag/wallabag.git
     cd wallabag
-    git checkout 2.1.1
-    SYMFONY_ENV=prod composer install --no-dev -o --prefer-dist
-    php bin/console wallabag:install --env=prod
+    ./install.sh
     php bin/console server:run --env=prod
```
Comment (j0k3r) [inline README.md]: You can remove the indentation here

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+TAG=$(git describe --tags $(git rev-list --tags --max-count=1))
+git checkout $TAG
```
Comment (j0k3r) [inline install.sh]: `--force`? To avoid error if user has changed some files in the repo?

Comment (nicosomb) [inline install.sh]: It's an installation, so no changed files I think

**Comment by tcitworld** [comment]
Instead of using an install.sh, could we this time use `make` ?
I could see : 
* `make install`: like `install.sh` 
* `make build` : run `grunt`
* `make tests` : run `ant prepare && phpunit`
* `make release` : produce a wallabag release

**Comment by nicosomb** [comment]
:+1: I'll look that tomorrow

**Comment by tcitworld** [comment]
I like your vision of tomorrow.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+clean: ## Clear the application cache
+	@rm -rf var/cache/*
+
+install: ## Install wallabag with the latest version
+	@sh scripts/install.sh $(ENV)
+
+update: ## Update the wallabag installation to the latest version
+	@sh scripts/update.sh $(ENV)
+
+run: ## Run the wallabag server
```
Comment (tcitworld) [inline Makefile]: build-in server

**Comment by tcitworld** [comment]
Don't forget to add that `make` is required now.

File: scripts/release.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
```
Comment (j0k3r) [inline scripts/release.sh]: Maybe you can put an example on how to launch that command (in comment)?

File: Makefile [hunk 5]
```diff
@@ -15,14 +17,14 @@ install: ## Install wallabag with the latest version
...
 
-run: ## Run the wallabag server
-	php bin/console server:run --env=$(ENV)
+run: ## Run the wallabag built-in server
+	@php bin/console server:run --env=$(ENV)
 
 build: ## Run grunt
 	@grunt
 
 test: ## Launch wallabag testsuite
```
Comment (tcitworld) [inline Makefile]: Maybe a regular `composer install`will be needed to install phpunit ?

Comment (nicosomb) [inline Makefile]: A `composer install` before launching testsuite?

Comment (tcitworld) [inline Makefile]: Hmm, this takes time.
Should be 
```
if vendor/phpunit/ not exists
     composer install
tests

File: RELEASE_PROCESS.md [hunk 6]
```diff
@@ -1,48 +1,44 @@
...
-- Create a new pull request with this title `DON'T MERGE Release wallabag 2.0.0-beta.1`. This pull request is used to launch builds on Travis-CI.
-- Run these commands to create the package:
+- Create a new pull request with this title `DON'T MERGE Release wallabag $LAST_WALLABAG_RELEASE`. This pull request is used to launch builds on Travis-CI.
+- Run these command to create the package:
 
 ```
-    git clone git@github.com:wallabag/wallabag.git -b release-2.0.0-beta.1 release-2.0.0-beta.1
-    SYMFONY_ENV=prod composer up -d=release-2.0.0-beta.1 --no-dev
-    tar czf wallabag-release-2.0.0-beta.1.tar.gz --exclude="var/*" --exclude=".git" release-2.0.0-beta.1
+make releas emaster /tmp wllbg-release prod
```
Comment (tcitworld) [inline RELEASE_PROCESS.md]: make release master

File: docs/de/user/installation.rst [hunk 7]
```diff
@@ -6,6 +6,19 @@ Voraussetzungen
 
 wallabag ist kompatibel mit PHP >= 5.5, inkl. PHP 7.
 
+.. note::
+Pour installer wallabag facilement, nous avons créé un ``Makefile``, vous avez donc besoin d'avoir installé l'outil ``make``.
```
Comment (tcitworld) [inline docs/de/user/installation.rst]: Why is there french here ?

File: docs/en/user/android.rst [hunk 8]
```diff
@@ -29,13 +29,13 @@ Fill in your wallabag data. You need to enter your wallabag address. It is impor
...
 
-After you have filled in your data, push the button Connection test and wait for the test to finish. 
+After you have filled in your data, push the button Connection test and wait for the test to finish.
 
 .. image:: ../../img/user/android_configuration_connection_test.en.png
     :alt: Connection test with your wallabag data
     :align: center
 
-The connection test shall finish with success. If not, you need to fix this first until you proceed. 
+The connection test shall finish with success. If not, you need to fix this first until you proceed.
```
Comment (tcitworld) [inline docs/en/user/android.rst]: shall -> should

File: docs/en/user/android.rst [hunk 9]
```diff
@@ -65,7 +65,7 @@ After hitting the save button, you get the following screen. The app proposes to
     :alt: Settings saved the first time
     :align: center
 
-Finally after the syncronisation finished successfully, you are presented the list of unread articles. 
+Finally after the syncronisation finished successfully, you are presented the list of unread articles.
```
Comment (tcitworld) [inline docs/en/user/android.rst]: you are presented to

File: docs/de/user/installation.rst (hunk 7 above)
Comment (nicosomb) [inline docs/de/user/installation.rst]: I don't know. It's not me. I'm sorry.

**Comment by tcitworld** [comment]
Please also add a page to explain the meaning of each parameter in `parameters.yml`

**Comment by nicosomb** [comment]
@tcitworld 😭

**Comment by nicosomb** [comment]
I added a draft for parameters explanation.

**Comment by j0k3r** [comment]
We should update this too:

- [ ] add step to enable redis/rabbitmq in async import
- [ ] fix link in import section in all languages (http://doc.wallabag.org/en/master/user/import.html)

**Comment by j0k3r** [comment]
Can we also add some doc about launching test in local? (requested from a user)

**Comment by nicosomb** [comment]
@j0k3r https://github.com/wallabag/wallabag/blob/fa4c39b37c1e1a1b52739a070775e9ba0fafe802/docs/en/developer/testsuite.rst ?

**Comment by j0k3r** [comment]
Awesome :trollface: (missed that commit)

**Pull request merged**